test/deploy/linux/cassandra/install/linux2/roles/configure/tasks/main.yml

```diff
@@ -66,5 +66,5 @@
   - name: Export HOSTNAME
     shell: "echo export NR_CLI_DB_HOSTNAME=localhost >> ~/.bashrc"
   - name: Export DB_PORT
-    shell: "echo export NR_CLI_DB_PORT=7199 >> ~/.bashrc"
```

Comment:
7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration